<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showProcess_Dialog</class>
 <widget class="QDialog" name="showProcess_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>当前正在运行的进程</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>输入你要附加的进程的PID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Button_attach">
       <property name="text">
        <string> 附加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_cancel">
       <property name="text">
        <string> 取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Button_cancel</sender>
   <signal>clicked()</signal>
   <receiver>showProcess_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
